scripts/release.js

```diff
@@ -156,6 +156,19 @@ logger.warn(`
 
 shouldContinuePrompt();
 
+// Creates a new React test app, install the component app as dependency and build the test app to ensure no errors or build issues
+process.chdir(path.join(__dirname, '..'));
+spawnOrFail('yalc', ['publish']);
+process.chdir(path.join(__dirname, '../..'));
+spawnOrFail('npx', ['create-react-app dependency-check-app']);
+process.chdir(path.join(__dirname, '../../dependency-check-app'));
+spawnOrFail('yalc', ['add amazon-chime-sdk-component-library-react']);
+spawnOrFail('npm', [`install amazon-chime-sdk-js@${updatedSdkVersion}`]);
+spawnOrFail('npm', ['install --save styled-components styled-system']);
+spawnOrFail('npm', ['run build']);
```

Comment:
Is there a way to validate the result?

Reply:
Do you mean verify the package version of component and js sdk? And also verify the build result?

Reply:
I meant check if there were any error during npm install.

Reply:
so you are asking if there is error with install command, what will be the behavior and how do we know if there are errors right? I have tested that if installed version is not available, the install step will be terminated with errors.